Packs/HostIo/Integrations/HostIo/hostio_test.py

```diff
@@ -0,0 +1,95 @@
+"""Base Integration for Cortex XSOAR - Unit Tests file"""
+
+import json
+import io
+
+
+def util_load_json(path):
+    with io.open(path, mode='r', encoding='utf-8') as f:
+        return json.loads(f.read())
+
+
+def test_domain(requests_mock):
+    """Tests the domain reputation command function.
+
+    Configures requests_mock instance to generate the appropriate
+    domain reputation API response, loaded from a local JSON file. Checks
+    the output of the command function with the expected output.
+    """
+    from hostio import Client, domain_command
+    from CommonServerPython import Common
+
+    domain_to_check = 'google.com'
+    mock_response = util_load_json('test_data/domain.json')
+    requests_mock.get(f'https://test.com/api/full/{domain_to_check}',
+                      json=mock_response)
+
+    client = Client(
+        base_url='https://test.com/api',
+        verify=False,
+        headers={
+            'Authorization': 'Bearer APIKEY'
+        }
+    )
+
+    args = {
+        'domain': domain_to_check,
+        'threshold': 65,
```

Comment:
you dont have a threshold argument in the command, please remove.

Reply:
done